andela/ah-django-unchained-frontend
# #162948806 Implement publishing of articles

#### What does this PR do?
Creates a button through which a user can be able to publish an article.

#### Description of Task to be completed?
* [x]    Configure Redux and thunk middleware
* [x]    Consume the `publish article` API endpoint

#### How should this be manually tested?
1. Clone this repository on your local machine using
   `git clone https://github.com/andela/ah-django-unchained-frontend.git`
2. Change your working directory to the project's directory using
   `cd ah-django-unchained-frontend`
3. Change to the specific branch `ft-create-draft-article-162948806 ` using
   `git checkout ft-publish-article-162948806`
4. Create a `.env` file and inside it add this line `REACT_APP_BASE_URL=app-base-url` where `app-base-url ` is the root url of the backend API
5. Run `npm install` to install all the modules listed in the `package.json` file.
6. Run `npm start` command to start the application.
7. The application should start successfully. On the browser, navigate to the URL `http://localhost:3000/createArticle`.
8. Create a draft article and save it.
9. Navigate to the created draft by clicking on `read more`
10. Click on the `Publish Article Button`

#### Link to PT story
[#162948806](https://www.pivotaltracker.com/story/show/162948806)
